src/ssh/azext_ssh/ssh_utils.py

```diff
@@ -9,14 +9,15 @@
 from knack import log
 from knack import util
 
-from . import file_utils
-
 logger = log.get_logger(__name__)
 
 
-def start_ssh_connection(ip, username, cert_file, private_key_file):
+def start_ssh_connection(port, ssh_args, ip, username, cert_file, private_key_file):
+    ssh_arg_list = []
+    if ssh_args:
+        ssh_arg_list = [ssh_args]
```

Comment:
Even if `ssh_args` was taken as `--ssh-args="-o SomeSshOption=3 -L 8080:localhost:8080"`, you will need to use [`shlex.split`](https://docs.python.org/3/library/shlex.html#shlex.split) to split the values into 

```py
>>> shlex.split('-o SomeSshOption=3 -L 8080:localhost:8080')
['-o', 'SomeSshOption=3', '-L', '8080:localhost:8080']
```

instead of simply wrapping `ssh_args` in a list.